src/components/LanguagePicker/index.js

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact';
+import I18n from 'i18nline';
+import style from './style';
+
+const locales = () => [
+	{ k: 'en', v: I18n.t('English') },
+	{	k: 'de', v: I18n.t('German') },
+	{	k: 'nl', v: I18n.t('Dutch') }
+];
+
+const LanguagePicker = props => (
```

Comment:
I think this should be a class component instead of splitting it out into an SFC + listener handler method.

Reply:
Yeah ok. Will change. Thanks!